corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -289,7 +289,9 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
   background-color: transparent;
   color: #685c53;
   justify-items: left;
-  padding: 10px 20px 0 20px;
+  <% if (model.isMultiSelect) { %>
+    margin: 10px 30px 0 10px;
+  <% } %>
```

Comment:
Is it intentional that this would remove the previously applied padding when it's not multi-select?